### PR TITLE
OF-1578 Monitoring small Graphs Not Being Generated.

### PR DESCRIPTION
**Pull request by annovanvliet**
Commented setter to null which is according to latest javadoc not allowed.

### CODE REVIEW

**Comment by guusdk** [comment]
Ah, that bugged me! Thanks for finding this! Could I tempt you to update the changelog et. al?

**Comment by annovanvliet** [comment]
Is there a issue-nr to which  could refer? There is a topic in discource https://discourse.igniterealtime.org/t/monitoring-small-graphs-not-being-generated/82194

File: src/plugins/monitoring/plugin.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <name>Monitoring Service</name>
     <description>Monitors conversations and statistics of the server.</description>
     <author>IgniteRealtime // Jive Software</author>
-    <version>1.6.1</version>
+    <version>1.6.2</version>
     <date>06/28/2018</date>
```
Comment (akrherz) [inline src/plugins/monitoring/plugin.xml]: please bump the release date too, thanks!

**Comment by akrherz** [comment]
whoa, I did something bad here with git.  Sorry Boggling.

**Comment by akrherz** [comment]
whew, awaiting CI before merging

**Pull request merged**